ballerina/llm_client_default.bal

```diff
@@ -47,7 +47,17 @@ public isolated distinct client class DefaultBallerinaAzureOpenAIModel {
         };
 
         http:Client cl = self.cl;
-        chat:CreateChatCompletionResponse chatResult = check cl->/chat/complete.post(chatBody);
+        http:Response chatResponse = check cl->/chat/complete.post(chatBody);
+        if (chatResponse.statusCode == 401 || chatResponse.statusCode == 403) {
```

Comment:
Extract status code out to a variable and reuse (below also).

Reply:
For below, since its range i guess we no need that